recipes/pkgconf/all/conanfile.py

```diff
@@ -139,21 +124,34 @@ def package_info(self):
             self.cpp_info.includedirs = []
             self.cpp_info.libdirs = []
 
-        bindir = os.path.join(self.package_folder, "bin")
-        self.output.info("Appending PATH env var: {}".format(bindir))
-        self.env_info.PATH.append(bindir)
-
         exesuffix = ".exe" if self.settings.os == "Windows" else ""
-        pkg_config = os.path.join(bindir, "pkgconf" + exesuffix).replace("\\", "/")
-        self.output.info("Setting PKG_CONFIG env var: {}".format(pkg_config))
+        pkg_config = os.path.join(self.package_folder, "bin", f"pkgconf{exesuffix}").replace("\\", "/")
+        self.conf_info.define("tools.gnu:pkg_config", pkg_config)
         self.buildenv_info.define_path("PKG_CONFIG", pkg_config)
-        self.env_info.PKG_CONFIG = pkg_config # remove in conan v2?
 
-        automake_extra_includes = tools.unix_path(os.path.join(self.package_folder , "bin", "aclocal").replace("\\", "/"))
-        self.output.info("Appending AUTOMAKE_CONAN_INCLUDES env var: {}".format(automake_extra_includes))
+        # TODO: replace by conan.tools.microsoft.unix_path when stateless
+        # (see https://github.com/conan-io/conan/issues/12192)
+        automake_extra_includes = tools_legacy.unix_path(os.path.join(self.package_folder , "bin", "aclocal").replace("\\", "/"))
```

Comment:
This should be in 1.54 ? https://github.com/conan-io/conan/pull/12178

Reply:
I'm not sure. What we need here is a conversion to unix_path of all paths vars coming from VirtualBuildEnv by conan in consumers recipes of pkgconfig running in win bash, and from my experimentations it doesn't work yet, or I'm missing something.

/cc @memsharded

Reply:
https://github.com/conan-io/conan/releases in beta8 and 1.57 `unix_path_package_info_legacy` was added that sounds like what was missing?